### PR TITLE
Add all recipes page to admin dashboard, create delete and edit actions

### PR DESCRIPTION
**Pull request by ErikasLau**
- Created All recipes page with all recipes created by all users in the website;
- All recipes page displays more information compared to My recipes page: Author, categories, ID;
- All recipes can be sorted by recipe name and creation date;
- Each recipe has two actions: edit and delete;
- Delete action works similary to action in my recipes page: input DELETE has to be typed to confirm deletion;
- Edit action allows to edit each recipe;
- Edit action redirects admin to admin recipe edit page (similar to recipe edit page in user dashboard) the page hower in independent and later can be updated to show or alow to edit more information;
- From My recipes page removed ID column and added average raiting.

### CODE REVIEW

File: Receptai.client/components/user/MyRecipes/RecipeContainer.vue [hunk 1]
```diff
@@ -22,11 +26,13 @@ const toBeDeleted = ref<number | null>(null);
 const recipeSelection = props.recipes?.map((recipe) => ({
   id: recipe.id,
   image: recipe.previewImage,
+  averageRaiting: recipe.averageRating,
   name: recipe.name,
   dateCreated: recipe.dateCreated.split("T")[0],
 }));
 
 const deleteRecipe = async () => {
+  console.log("Delete");
```
Comment (svetikas) [inline Receptai.client/components/user/MyRecipes/RecipeContainer.vue]: left a console.log

File: Receptai.client/components/user/MyRecipes/RecipeContainer.vue [hunk 2]
```diff
@@ -22,11 +26,13 @@ const toBeDeleted = ref<number | null>(null);
 const recipeSelection = props.recipes?.map((recipe) => ({
   id: recipe.id,
   image: recipe.previewImage,
+  averageRaiting: recipe.averageRating,
```
Comment (svetikas) [inline Receptai.client/components/user/MyRecipes/RecipeContainer.vue]: misspelling

File: Receptai.client/components/admin/AllRecipes/RecipeContainer.vue [hunk 3]
```diff
@@ -0,0 +1,171 @@
+<script setup lang="ts">
+import axios from "axios";
+import type {
+  UserCookie,
+  Recipe,
+  RecipeContainerColumn,
+} from "@/typescript/types";
+import { addNotification } from "@/store/store";
+import dateWithTime from "~/typescript/dateFormating";
```
Comment (svetikas) [inline Receptai.client/components/admin/AllRecipes/RecipeContainer.vue]: čia šitie importai su relative, ~ ir @ jau gasdina

**Comment by svetikas** [comment]
Basic statinę analizę atlikau, prašau peržiūrėti komentarus ir atsakyti.

File: Receptai.client/components/admin/AllRecipes/RecipeContainer.vue (hunk 3 above)
Comment (ErikasLau) [reply Receptai.client/components/admin/AllRecipes/RecipeContainer.vue]: Jo, patvarkysiu, kažkodėl man defaultina į tilda

File: Receptai.client/components/user/MyRecipes/RecipeContainer.vue (hunk 2 above)
Comment (ErikasLau) [reply Receptai.client/components/user/MyRecipes/RecipeContainer.vue]: Ups

File: Receptai.client/components/user/MyRecipes/RecipeContainer.vue (hunk 1 above)
Comment (ErikasLau) [reply Receptai.client/components/user/MyRecipes/RecipeContainer.vue]: Bus panaikinta

**Pull request merged**